certauth.go

```diff
@@ -77,6 +80,22 @@ func (a *Auth) Handler(h http.Handler) http.Handler {
 	})
 }
 
+// RouterHandler implements the httprouter.Handle for integration with github.com/julienschmidt/httprouter
+func (a *Auth) RouterHandler(h httprouter.Handle) httprouter.Handle {
```

Comment:
I am conflicted on this as I think you can just wrap this in the httphandler.HandlerFunc from the caller vs us having to do this and then add the dep to httprouter.  

```
router.HandlerFunc(auth.Hande)